diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 2a60ac262584a6cdae0dbdc3e73058be914c0f6c..0d0ceb22ecd686f4f86e757abaecacd0a38a47ae 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -176,6 +176,8 @@ exports.networks = [
     { chainId: 6322n, shortName: 'aura' },
     { chainId: 6398n, shortName: 'connext-sepolia' },
     { chainId: 6688n, shortName: 'iris' },
+    { chainId: 6900n, shortName: 'cataclysm-1' },
+    { chainId: 6911n, shortName: 'nibiru-testnet-2' },
     { chainId: 7000n, shortName: 'zetachain-mainnet' },
     { chainId: 7001n, shortName: 'zetachain-testnet' },
     { chainId: 7171n, shortName: 'bitrock' },
